.github/workflows/run-tests.yml

```diff
@@ -105,21 +103,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-22.04, macos-13, windows-latest ]
-        java: [ 8, 11, 17, 21 ]
-        exclude:
-          # exclude non-java 8 on macos and windows builds
-          - os: macos-13
-            java: 11
-          - os: windows-latest
-            java: 11
-          - os: macos-13
-            java: 17
```

Comment:
I think we can keep the exclusion for Windows/MacOSX on java 17. Just running the tests in a single java for them is ok.